README.md

```diff
@@ -23,22 +23,91 @@ From your Terrier directory, you should:
 1. edit the terrier.properties file to specify the query expansion models in the querying.processes list
 
 ```
-querying.processes=terrierql:TerrierQLParser,parsecontrols:TerrierQLToControls,parseql:TerrierQLToMatchingQueryTerms,matchopql:MatchingOpQLParser,applypipeline:ApplyTermPipeline,localmatching:LocalManager$ApplyLocalMatching,rm1:RM1,rm3:RM3,ax:AxiomaticQE,qe:QueryExpansion,labels:org.terrier.learning.LabelDecorator,filters:LocalManager$PostFilterProcess'
+querying.processes=terrierql:TerrierQLParser,parsecontrols:TerrierQLToControls,parseql:TerrierQLToMatchingQueryTerms,matchopql:MatchingOpQLParser,applypipeline:ApplyTermPipeline,localmatching:LocalManager$ApplyLocalMatching,ax:AxiomaticQE,qe:QueryExpansion,labels:org.terrier.learning.LabelDecorator,filters:LocalManager$PostFilterProcess'
 
 ```
 
 2. Invoke `batchretrieval` or `interactive` command while specifying the relevant controls, and the terrier-prf package
 
-```
-bin/terrier br -w BM25 -c rm1:on -o ./bm25.rm1.res -P org.terrier:terrier-prf:0.1-SNAPSHOT
+```shell
+
+bin/terrier br -w BM25 -c axqe:on -o ./bm25.axqe.res -P org.terrier:terrier-prf
 
-bin/terrier br -w BM25 -c rm3:on -o ./bm25.rm3.res -P org.terrier:terrier-prf:0.1-SNAPSHOT
+```
 
-bin/terrier br -w BM25 -c axqe:on -o ./bm25.axqe.res -P org.terrier:terrier-prf:0.1-SNAPSHOT
+## Use of AxiomaticQE in PyTerrier
+
+AxiomaticQE can be used in PyTerrier.
+
+```python
+from pyterrier.terrier.rewrite import QueryExpansion
+@pt.java.required
+class AxiomaticQE(QueryExpansion):
+    '''
+        Performs query expansion using axiomatic query expansion.
+
+        This transformer must be followed by a Terrier Retrieve() transformer.
+        The original query is saved in the `"query_0"` column, which can be restored using `pt.rewrite.reset()`.
+
+        Instance Attributes:
+         - fb_terms(int): number of feedback terms. Defaults to 10
+         - fb_docs(int): number of feedback documents. Defaults to 3
+    '''
+    def __init__(self, *args, fb_terms=10, fb_docs=3, **kwargs):
+        """
+        Args:
+            index_like: the Terrier index to use
+            fb_terms(int): number of terms to add to the query. Terrier's default setting is 10 expansion terms.
+            fb_docs(int): number of feedback documents to consider. Terrier's default setting is 3 feedback documents.
+        """
+        rm = pt.java.autoclass("org.terrier.querying.AxiomaticQE")
+        self.fb_terms = fb_terms
+        self.fb_docs = fb_docs
+        kwargs["qeclass"] = rm
+        super().__init__(*args, **kwargs)
+
+    def transform(self, queries_and_docs):
+        self.qe.fbTerms = self.fb_terms
+        self.qe.fbDocs = self.fb_docs
+        return super().transform(queries_and_docs)
+```
 
+Example usage
+```python

```

Comment:
need to add the prf package, something like:

```python
pt.java.add_package('com.github.terrierteam', 'terrier-prf', '-SNAPSHOT')
```

(Haven't tested this specifically)